src/qml/QFieldSettings.qml

```diff
@@ -67,6 +68,15 @@ Page {
           description: qsTr( "If disabled, the mouse will act as a stylus pen." )
           settingAlias: "mouseAsTouchScreen"
       }
+      Component.onCompleted: {
+          var i;
+          for (i = 0; i < count; i++) {
+              if (get(i).settingAlias == 'nativeCamera')
+                  setProperty(i, 'isVisible', platformUtilities.supportsNativeCamera)
+              else
+                  setProperty(i, 'isVisible', true)
+          }
+      }
   }
```

Comment:
```suggestion
      Component.onCompleted: {
          for (var i = 0; i < settingsModel.count; i++) {
              if (settingsModel.get(i).settingAlias === 'nativeCamera') {
                  settingsModel.setProperty(i, 'isVisible', platformUtilities.supportsNativeCamera)
              } else {
                  settingsModel.setProperty(i, 'isVisible', true)   
              }
          }
      }
  }
```

Reply:
@m-kuhn I would write it like this:
1) curly brackets in js are cheap, since they are the same line as the opening statement
2) always use `===` instead of `==`
3) `for (var i = 0; i < l; i++ )` is valid syntax, no need to predefine the `var i`. The only problem with `var` is it is not deleted after the loop, but it is fine.
4) warning, opinion: I think we should not abuse the implicit usage of the QML element's properties and methods. When looking at your diff, I was a bit confused where this `count` variable comes from and I had to check the whole file and context. Now it is clear it is coming from the model. The same is valid for `get` and `setProperty`.

Reply:
Agreed on each point. Except for 3, where I think that the "problem" is actually a feature :+1: